src/models/Config.php

```diff
@@ -314,6 +314,8 @@ public function destroy(): bool
         $sql = 'DELETE FROM config';
         $req = $this->Db->prepare($sql);
         $this->Db->execute($req);
-        return (bool) $this->create();
+        $this->create();
```

Comment:
what if create would return bool (I don't see why the req->execute is cast to int...)? Then you store the result of create and return that.